README.md

```diff
@@ -18,9 +18,14 @@ This layer depends on:
 * poky (http://git.yoctoproject.org/cgit/cgit.cgi/poky)
 * meta-openembedded (http://cgit.openembedded.org/meta-openembedded/)
 * meta-virtualization (http://git.yoctoproject.org/cgit/cgit.cgi/meta-virtualization),
-if the "process-containment" distro feature has been enabled
+  if the "process-containment" distro feature has been enabled
 * meta-bistro (https://github.com/Pelagicore/meta-bistro)
 
+The relationship with meta-bistro, which is also developed at Pelagicore, is
+that meta-bistro contains general software recipes, but no distribution and no
```

Comment:
I would add that `meta-bistro` adds IVi and development related recipes and changes to the PELUX image. For instance it adds the development tools packageconfig and the ivi packageconfig (have to verify this).

Reply:
meta-bistro does not change the PELUX image. Rather, the PELUX image makes use of things defined in meta-bistro.

I just checked, and it does create/append to some packagegroups, but the relation is still that PELUX  uses what's defined in bistro.

Reply:
That said, I think the comment about describing what it is that bistro brings rather than generic software, is a good one :)